src/objects/Mesh.js

```diff
@@ -5,7 +5,7 @@
  * @author jonobr1 / http://jonobr1.com/
  */
 
-THREE.Mesh = function ( geometry, material ) {
+THREE.Mesh = function ( geometry, material, mode ) {
```

Comment:
I wouldn't add `mode` here. I would prefer to keep constructors simple and use this sort of API instead:

```javascript
new THREE.Mesh( geometry, material ).setMode( THREE.TriangleFan )
```